lib/ssl/test/ssl_bench_test_lib.erl

```diff
@@ -47,7 +47,7 @@ setup(Name) ->
             PeerOptions =
                 #{name => NameStr,
                   host => Host},
-            ?CT_LOG("PeerOptions: ~p~n", [PeerOptions]),
+            %% io:format("PeerOptions: ~p~n", [PeerOptions]),
```

Comment:
Why are we doing io:format here in this commented code?
I think we should be consistent through the file. Question is if we should keep it at all? Although it bothers me less in test code, I think in the normal case we do not want commented out code.

Reply:
This is just a mistake and should not be included, when running measurements 1000 times you don't want to bother
with starting common tests and look at webpages.